### ISSUE TITLE
Animation Nodes Error Message- Did I break it?

### ISSUE THREAD

**Issue by nvasquez1623** [JacquesLucke/animation_nodes#562]
Hello  I installed the  latest version of animation nodes in 2.77a.  I saw the tools / buttons in the panel but did not use it.  I closed blender then reopened a while later.  The add on appeared to be there in the user panel but I no longer saw the  buttons.  When I disabled then re-enabled I got an error message. (see attached)

Thanks for all of the work you have put in to this and everything else you have done for Blender.  If you need more info let me know thanks

-Nick
![animation nodes error](https://cloud.githubusercontent.com/assets/19726827/15773758/ae732174-292d-11e6-944a-7121cd837da8.PNG)

**Comment by JacquesLucke** [comment]
oh damn it! that really looks like a bug!
I wonder why this never happened to me lately.

can you make sure that No old version of AN is installed?
(best thing would be to go into the addons folder, remove animation nodes and reinstall it again)

I will check in more detail as soon as I got access to a pc in a few hours!

**Comment by nvasquez1623** [comment]
I will do what you suggest.  I'm on the U.S. west coast, 2:30am here, So I will do that in the morning.  I'll keep you updated.

-Nick

**Comment by JacquesLucke** [comment]
okay, thx for the report btw :)

**Comment by nvasquez1623** [comment]
Okay I fixed my problem.  Probalby did it the hard way, but that's okay.  Long story short I ended up deleting/ reinstalling both Blender 2.77a & Animation Nodes.

What I think I did:  I think I deleted the Animation nodes folder.   But it still showed up in the addons in Blender.  everything appears to be there now.  I will test it a bit later.

Hopefully it was just my poor form and not much else.

Regards,
-Nick
